python/mxnet/contrib/onnx/onnx2mx/_op_translations.py

```diff
@@ -407,8 +407,22 @@ def cast(attrs, inputs, proto_obj):
 
 def split(attrs, inputs, proto_obj):
     """Splits an array along a particular axis into multiple sub-arrays."""
+    split_list = attrs.get('split') if 'split' in attrs else []
     new_attrs = translation_utils._fix_attribute_names(attrs,
                                                        {'split' : 'num_outputs'})
+    if 'axis' not in attrs:
+        new_attrs = translation_utils._add_extra_attributes(new_attrs, {'axis': 0})
+
+    if not split_list:
+        num_outputs = len(proto_obj.model_metadata.get('output_tensor_data'))
```

Comment:
Can this be empty?

Reply:
'output_tensor_data' is basically outputs field of ONNX operators. here: https://github.com/onnx/onnx/blob/master/docs/Operators.md#split outputs.
If its empty, onnx graph checker wont pass. It will be invalid onnx graph.
So, It shouldn't be empty.